backend/src/modules/auth/authRoutes.js

```diff
@@ -165,6 +169,32 @@ router.put(
       return res.status(422).json({ error: '422: Not existing user' });
     }
 
+    const userDetail = await dbConnection.query(
+      'SELECT email FROM users WHERE user_id = ? AND active = true;',
+      [userId]
+    );
+
+    if (userDetail.length > 0){
```

Comment:
Tohle bude vzdycky platit, protoze o par radku vys updatujes usera s timhle ID uz tam je check, zda tohle ID existuje. Takze nasledne kdyz selectnes toho usera, tak tam vzdycky bude.

Reply:
ale co když právě že neselectnu, takový uživatel v DB nebude? pak mi to lehne tady ne? 
`const { email } = userDetail[0];`